server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/main/java/org/apache/james/webadmin/vault/routes/DeletedMessagesVaultRoutes.java

```diff
@@ -72,11 +73,12 @@
 @Produces(Constants.JSON_CONTENT_TYPE)
 public class DeletedMessagesVaultRoutes implements Routes {
 
-    enum UserVaultAction {
+    enum VaultAction {
         RESTORE("restore"),
-        EXPORT("export");
+        EXPORT("export"),
+        PURGE("purge");
```

Comment:
This action should be defined in a different enum. Please keep the UserVaultAction, and create a new enum GlobalVaultAction. Because purge(globally) doesn't share same scope with restore or export (user-based)

Reply:
IMO, `action` in DMVRoutes have any conflict for `user` or not. We dont' need to write the same enum for the `action`

Reply:
I find current implementation acceptable. I don't see the value added by having two set of enums.